test/panel.test.tsx

```diff
@@ -55,7 +55,33 @@ describe('Panel', () => {
     expect(panelWrapper.find(Icon)).toHaveLength(1)
     expect(bodyWrapper.find(Icon)).toHaveLength(1)
   })
+  it('Panel collapse on header click', () => {
```

Comment:
per the convention of the tests, could we rename the test to: 

`it('should collapse the panel on header click')`